manifests/application.pp

```diff
@@ -17,11 +17,13 @@
   Optional[String] $proxy_username = $::katello::proxy_username,
   Optional[String] $proxy_password = $::katello::proxy_password,
 ) {
+  include ::foreman
   include ::certs
   include ::certs::apache
   include ::certs::foreman
   include ::certs::pulp_client
 
+  $post_sync_url = "${$::foreman::foreman_url}${deployment_url}/api/v2/repositories/sync_complete?token=${post_sync_token}"
```

Comment:
I'm surprised this works. Usually we have ```${::foreman::foreman_url}```. The tests appear to pass so this is valid syntax?

Reply:
Interesting, it must be. I was actually a typo so I've update to match the rest of our code base.